<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Interface</class>
 <widget class="QWidget" name="Interface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Interface</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_5">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="_sizeLabel">
       <property name="text">
        <string>Target size</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="_w">
         <property name="text">
          <string>W:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="_hSpinBox"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="_h">
         <property name="text">
          <string>H:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="_wSpinBox"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Target type</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QRadioButton" name="_squareRadio">
          <property name="text">
           <string>Square</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="_rectangleRadio">
          <property name="text">
           <string>Rectangle</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="_circleRadio">
          <property name="text">
           <string>Сircle</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="_ellipseRadio">
          <property name="text">
           <string>Ellipse</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="_a">
         <property name="text">
          <string>А:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="_aSpinBox">
         <property name="maximum">
          <number>9999</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="_b">
         <property name="text">
          <string>B:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="_bSpinBox">
         <property name="maximum">
          <number>9999</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Check type</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QRadioButton" name="_regularRadio">
          <property name="text">
           <string>Regular</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="_randomRadio">
          <property name="text">
           <string>Random</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="_check">
       <property name="text">
        <string>Check</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="_result">
       <property name="text">
        <string>Result:</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGraphicsView" name="graphicsView"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
